route.go

```diff
@@ -31,6 +32,7 @@ func (ep forwardEndpoint) Handler(prefix string, templates *template.Template, c
 	rp.Transport = &http.Transport{
 		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
 	}
+	rp.FlushInterval = 200 * time.Millisecond
```

Comment:
To fast?

Reply:
That's a good question - I'd like to pull this out into a constant at the top of the file, so it's clear that we have a tunable parameter here that we need to be aware of.

Reply:
Yeap that sounds good